debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SourceInfo.java

```diff
@@ -108,4 +84,19 @@ public String toString() {
         map.put(TIMESTAMP_KEY, tsMicro);
         return map.toString();
     }
+
+    @Override
+    protected Instant timestamp() {
+        return Instant.ofEpochSecond(tsMicro/1000);
+    }
+
+    @Override
+    protected SnapshotRecord snapshot() {
+        return snapshot ? SnapshotRecord.TRUE : SnapshotRecord.FALSE;
+    }
+
+    @Override
+    protected String database() {
+        return "NULL";
```

Comment:
Could you add a clarifying comment; is there no notion of "database" in Cassandra?

Reply:
> Could you add a clarifying comment; is there no notion of "database" in Cassandra?

Got it, added clarifying comment accordingly.